app/grandchallenge/serving/urls.py

```diff
@@ -41,4 +42,15 @@ def to_url(self, value):
         ),
         serve_submissions,
     ),
+    path(
+        (
+            f"{settings.COMPONENTS_FILES_SUBDIRECTORY}/"
+            f"componentinterfacevalue/"
+            f"<uuidprefix:pa>/"
```

Comment:
Are we really looking for `uuidprefix` types here? I don't think that this matches the definition in `component_interface_value_path`

Reply:
Should I add a converter here for the `pk_as_padded_hex` that is used in the model?

Reply:
Renamed it to more generic prefix converter. If you'd still rather just want comments, I'll change it back